DOCS.md

```diff
@@ -715,6 +716,29 @@ login({appState: JSON.parse(fs.readFileSync('appstate.json', 'utf8'))}, (err, ap
 
 ---------------------------------------
 
+<a name="validThreadColors"></a>
+### api.validThreadColors
+
+Returns a dictionary mapping names of all currently valid thread colors to their hexadecimal values that are accepted by [`api.changeThreadColor`](#changeThreadColor). These colors, listed below, are the ones present in the palette UI used for selecting thread colors on the Messenger client. Due to Facebook backend changes, the thread color can no longer be set to an arbitrary hex value.
```

Comment:
Maybe we can move the note about "Due to Facebook backend changes, the thread color can no longer be set to an arbitrary hex value." to the CHANGELOG.md file and put a link there to this PR?  (See the other entries for an example)